### ISSUE TITLE
Grafana: Drives show up in "statistics" and "overview" but not in "drives"

### ISSUE THREAD

**Issue by e-tplus** [adriankumpf/teslamate#1911]
Some drives do not show up in "Drives" but they are listed in "Statistics" and show up in the "Overview" timeline. In statistics these drives are lacking all data but they show up in the map. 

Seems car was offline between the drives. 

```
teslamate_1  | 2021-08-16 17:06:49.536 car_id=1 [info] Start / :online
teslamate_1  | 2021-08-16 17:06:49.581 car_id=1 [info] Connecting ...
teslamate_1  | 2021-08-16 17:07:03.340 car_id=1 [info] Driving / Start
teslamate_1  | 2021-08-16 17:12:41.534 [warn] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5/vehicle_data -> 504 (10092.641 ms)
teslamate_1  | 2021-08-16 17:12:41.534 car_id=1 [error] Error / :timeout
teslamate_1  | 2021-08-16 17:14:06.146 [warn] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5/vehicle_data -> 503 (294.495 ms)
teslamate_1  | 2021-08-16 17:14:06.147 [warn] TeslaApi.Error / "upstream connect error or disconnect/reset before h
eaders. reset reason: connection failure"
teslamate_1  | 2021-08-16 17:14:06.147 car_id=1 [error] Error / :unknown
teslamate_1  | 2021-08-16 17:15:25.596 [error] ** (RuntimeError) Client Error: %{"error_type" => "client_error", "m
sg_type" => "data:error", "tag" => "1052137850", "value" => "owner_api error: `Post \"https://ownerapi-internal.vn.
teslamotors.com:443/authenticate/vehicles/1052137850?operation=streaming_authenticate\": context deadline exceeded 
(Client.Timeout exceeded while awaiting headers)`"}
teslamate_1  |     (teslamate 1.23.7) lib/tesla_api/stream.ex:150: TeslaApi.Stream.handle_frame/2
teslamate_1  |     (websockex 0.4.3) lib/websockex.ex:1113: WebSockex.try_callback/3
teslamate_1  |     (websockex 0.4.3) lib/websockex.ex:797: WebSockex.common_handle/4
teslamate_1  |     (stdlib 3.15.1) proc_lib.erl:226: :proc_lib.init_p_do_apply/3
teslamate_1  | 
teslamate_1  | 2021-08-16 17:15:25.982 [warn] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5 -> 503 (295.737 ms)
teslamate_1  | 2021-08-16 17:15:25.982 [warn] TeslaApi.Error / "upstream connect error or disconnect/reset before h
eaders. reset reason: connection failure"
teslamate_1  | 2021-08-16 17:15:25.982 car_id=1 [error] Error / :unknown
teslamate_1  | 2021-08-16 17:16:04.712 car_id=1 [info] Start / :online
teslamate_1  | 2021-08-16 17:16:04.719 car_id=1 [info] Connecting ...
teslamate_1  | 2021-08-16 17:16:04.720 car_id=1 [info] Driving / Start
teslamate_1  | 2021-08-16 17:16:36.556 [warn] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5/vehicle_data -> 504 (10523.737 ms)
teslamate_1  | 2021-08-16 17:16:36.557 car_id=1 [error] Error / :timeout
teslamate_1  | 2021-08-16 17:19:34.031 [warn] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5/vehicle_data -> 503 (295.300 ms)
teslamate_1  | 2021-08-16 17:19:34.031 [warn] TeslaApi.Error / "upstream connect error or disconnect/reset before h
eaders. reset reason: connection failure"
teslamate_1  | 2021-08-16 17:19:34.031 car_id=1 [error] Error / :unknown
teslamate_1  | 2021-08-16 17:20:28.925 [warn] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5/vehicle_data -> 503 (293.625 ms)
teslamate_1  | 2021-08-16 17:20:28.925 [warn] TeslaApi.Error / "upstream connect error or disconnect/reset before h
eaders. reset reason: connection failure"
teslamate_1  | 2021-08-16 17:20:28.925 car_id=1 [error] Error / :unknown
teslamate_1  | 2021-08-16 17:22:20.420 car_id=1 [info] Received stream data: :inactive
teslamate_1  | 2021-08-16 17:22:40.433 [error] ** (RuntimeError) Client Error: %{"error_type" => "client_error", "m
sg_type" => "data:error", "tag" => "1052137850", "value" => "owner_api error: `Got HTTP status: 503 Service Unavail
able for . Response: upstream connect error or disconnect/reset before headers. reset reason: connection failure`"}
teslamate_1  |     (teslamate 1.23.7) lib/tesla_api/stream.ex:150: TeslaApi.Stream.handle_frame/2
teslamate_1  |     (websockex 0.4.3) lib/websockex.ex:1113: WebSockex.try_callback/3
teslamate_1  |     (websockex 0.4.3) lib/websockex.ex:797: WebSockex.common_handle/4
teslamate_1  |     (stdlib 3.15.1) proc_lib.erl:226: :proc_lib.init_p_do_apply/3
teslamate_1  | 
teslamate_1  | 2021-08-16 17:23:03.425 [warn] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5/vehicle_data -> 504 (19080.711 ms)
teslamate_1  | 2021-08-16 17:23:03.426 car_id=1 [error] Error / :timeout
teslamate_1  | 2021-08-16 17:23:34.331 car_id=1 [info] Start / :online
teslamate_1  | 2021-08-16 17:23:34.337 car_id=1 [info] Connecting ...
teslamate_1  | 2021-08-16 17:23:34.338 car_id=1 [info] Driving / Start
teslamate_1  | 2021-08-16 17:24:10.367 [warn] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5/vehicle_data -> 504 (10093.714 ms)
teslamate_1  | 2021-08-16 17:24:30.679 [warn] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5/vehicle_data -> 504 (10310.252 ms)
teslamate_1  | 2021-08-16 17:24:30.679 car_id=1 [error] Error / :timeout
teslamate_1  | 2021-08-16 17:34:21.715 [warn] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5/vehicle_data -> 504 (16767.991 ms)
teslamate_1  | 2021-08-16 17:34:21.715 car_id=1 [error] Error / :timeout
teslamate_1  | 2021-08-16 17:34:46.753 [error] ** (RuntimeError) Client Error: %{"error_type" => "client_error", "m
sg_type" => "data:error", "tag" => "1052137850", "value" => "owner_api error: `Post \"https://ownerapi-internal.vn.
teslamotors.com:443/authenticate/vehicles/1052137850?operation=streaming_authenticate\": context deadline exceeded 
(Client.Timeout exceeded while awaiting headers)`"}
teslamate_1  |     (teslamate 1.23.7) lib/tesla_api/stream.ex:150: TeslaApi.Stream.handle_frame/2
teslamate_1  |     (websockex 0.4.3) lib/websockex.ex:1113: WebSockex.try_callback/3
teslamate_1  |     (websockex 0.4.3) lib/websockex.ex:797: WebSockex.common_handle/4
teslamate_1  |     (stdlib 3.15.1) proc_lib.erl:226: :proc_lib.init_p_do_apply/3
teslamate_1  | 
teslamate_1  | 2021-08-16 17:34:58.571 [info] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5/vehicle_data -> 408 (9566.565 ms)
teslamate_1  | 2021-08-16 17:34:58.571 car_id=1 [error] Error / :vehicle_unavailable
teslamate_1  | 2021-08-16 17:35:42.446 [warn] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5/vehicle_data -> 504 (10431.034 ms)
teslamate_1  | 2021-08-16 17:35:42.446 car_id=1 [error] Error / :timeout
teslamate_1  | 2021-08-16 17:36:12.876 car_id=1 [info] Start / :offline
teslamate_1  | 2021-08-16 17:37:04.727 [warn] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5 -> 503 (240.255 ms)
teslamate_1  | 2021-08-16 17:37:04.727 [warn] TeslaApi.Error / "upstream connect error or disconnect/reset before h
eaders. reset reason: connection failure"
teslamate_1  | 2021-08-16 17:37:04.727 car_id=1 [error] Error / :unknown
teslamate_1  | 2021-08-16 17:48:30.488 [warn] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5 -> 503 (293.196 ms)
teslamate_1  | 2021-08-16 17:48:30.488 [warn] TeslaApi.Error / "upstream connect error or disconnect/reset before h
eaders. reset reason: connection failure"
teslamate_1  | 2021-08-16 17:48:30.488 car_id=1 [error] Error / :unknown
teslamate_1  | 2021-08-16 17:57:59.474 [info] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5/vehicle_data -> 408 (210.459 ms)
teslamate_1  | 2021-08-16 17:57:59.475 car_id=1 [error] Error / :vehicle_unavailable
teslamate_1  | 2021-08-16 17:59:41.369 [info] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5/vehicle_data -> 408 (9201.006 ms)
teslamate_1  | 2021-08-16 17:59:41.369 car_id=1 [error] Error / :vehicle_unavailable
teslamate_1  | 2021-08-16 18:00:20.915 [info] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5/vehicle_data -> 408 (9338.905 ms)
teslamate_1  | 2021-08-16 18:00:20.915 car_id=1 [error] Error / :vehicle_unavailable
teslamate_1  | 2021-08-16 18:01:01.657 [warn] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5/vehicle_data -> 504 (10251.494 ms)
teslamate_1  | 2021-08-16 18:01:01.657 car_id=1 [error] Error / :timeout
teslamate_1  | 2021-08-16 18:01:42.009 [warn] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5/vehicle_data -> 504 (10193.990 ms)
teslamate_1  | 2021-08-16 18:01:42.010 car_id=1 [error] Error / :timeout
teslamate_1  | 2021-08-16 18:02:22.491 [warn] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5/vehicle_data -> 504 (10097.765 ms)
teslamate_1  | 2021-08-16 18:02:22.491 car_id=1 [error] Error / :timeout
teslamate_1  | 2021-08-16 18:03:08.512 car_id=1 [info] Start / :online
teslamate_1  | 2021-08-16 18:03:08.520 car_id=1 [info] Connecting ...
teslamate_1  | 2021-08-16 18:03:47.884 [info] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5/vehicle_data -> 408 (9360.896 ms)
teslamate_1  | 2021-08-16 18:03:49.340 car_id=1 [warn] Discarded incomplete fetch result
teslamate_1  | 2021-08-16 18:04:08.606 [info] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5/vehicle_data -> 408 (9263.944 ms)
teslamate_1  | 2021-08-16 18:04:09.110 car_id=1 [warn] Discarded incomplete fetch result
teslamate_1  | 2021-08-16 18:04:29.333 [warn] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5/vehicle_data -> 504 (10221.454 ms)
teslamate_1  | 2021-08-16 18:04:29.333 car_id=1 [error] Error / :timeout
teslamate_1  | 2021-08-16 18:04:58.975 [info] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5/vehicle_data -> 408 (9640.458 ms)
teslamate_1  | 2021-08-16 18:04:59.109 car_id=1 [warn] Discarded incomplete fetch result
teslamate_1  | 2021-08-16 18:05:18.931 [info] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5/vehicle_data -> 408 (9821.199 ms)
teslamate_1  | 2021-08-16 18:05:19.720 car_id=1 [warn] Discarded incomplete fetch result
teslamate_1  | 2021-08-16 18:05:39.903 [warn] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5/vehicle_data -> 504 (10182.579 ms)
teslamate_1  | 2021-08-16 18:05:39.903 car_id=1 [error] Error / :timeout
teslamate_1  | 2021-08-16 18:05:51.723 [error] Vehicle Error: Vehicle is offline
teslamate_1  | 2021-08-16 18:06:05.062 [info] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5/vehicle_data -> 408 (5157.072 ms)
teslamate_1  | 2021-08-16 18:06:18.440 car_id=1 [info] Start / :offline
teslamate_1  | 2021-08-16 18:06:18.463 car_id=1 [info] Disconnecting ...
teslamate_1  | 2021-08-16 18:41:53.060 car_id=1 [info] Start / :online
teslamate_1  | 2021-08-16 18:41:53.069 car_id=1 [info] Connecting ...
teslamate_1  | 2021-08-16 18:41:53.070 car_id=1 [info] Driving / Start
teslamate_1  | 2021-08-16 18:43:28.228 [warn] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5/vehicle_data -> 504 (10636.401 ms)
teslamate_1  | 2021-08-16 18:43:28.228 car_id=1 [error] Error / :timeout
teslamate_1  | 2021-08-16 18:43:47.397 [info] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5/vehicle_data -> 408 (9167.368 ms)
teslamate_1  | 2021-08-16 18:43:47.564 car_id=1 [warn] Discarded incomplete fetch result
teslamate_1  | 2021-08-16 18:50:17.992 [info] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5/vehicle_data -> 408 (9115.766 ms)
teslamate_1  | 2021-08-16 18:50:18.171 car_id=1 [warn] Discarded incomplete fetch result
teslamate_1  | 2021-08-16 18:50:37.702 [info] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5/vehicle_data -> 408 (9528.955 ms)
teslamate_1  | 2021-08-16 18:50:38.324 car_id=1 [warn] Discarded incomplete fetch result
teslamate_1  | 2021-08-16 18:50:57.621 [info] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5/vehicle_data -> 408 (9296.769 ms)
teslamate_1  | 2021-08-16 18:50:57.746 car_id=1 [warn] Discarded incomplete fetch result
teslamate_1  | 2021-08-16 18:51:04.048 car_id=1 [info] Received stream data: :inactive
teslamate_1  | 2021-08-16 18:51:07.904 [info] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5/vehicle_data -> 408 (156.187 ms)
teslamate_1  | 2021-08-16 18:51:08.008 car_id=1 [warn] Vehicle went offline while driving
teslamate_1  | 2021-08-16 18:51:15.643 car_id=1 [info] Vehicle is back online
teslamate_1  | 2021-08-16 19:00:20.174 [info] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5/vehicle_data -> 408 (9204.692 ms)
teslamate_1  | 2021-08-16 19:00:20.641 car_id=1 [warn] Discarded incomplete fetch result
teslamate_1  | 2021-08-16 19:00:40.712 [warn] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5/vehicle_data -> 504 (10070.383 ms)
teslamate_1  | 2021-08-16 19:00:40.712 car_id=1 [error] Error / :timeout
teslamate_1  | 2021-08-16 19:02:47.602 [info] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5/vehicle_data -> 408 (9722.602 ms)
teslamate_1  | 2021-08-16 19:02:47.957 car_id=1 [warn] Discarded incomplete fetch result
teslamate_1  | 2021-08-16 19:03:07.377 [info] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5/vehicle_data -> 408 (9418.479 ms)
teslamate_1  | 2021-08-16 19:03:07.490 car_id=1 [warn] Discarded incomplete fetch result
teslamate_1  | 2021-08-16 19:03:11.791 car_id=1 [info] Received stream data: :inactive
teslamate_1  | 2021-08-16 19:03:28.356 [warn] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5/vehicle_data -> 504 (10864.355 ms)
teslamate_1  | 2021-08-16 19:03:28.356 car_id=1 [error] Error / :timeout
teslamate_1  | 2021-08-16 19:03:48.672 [warn] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5/vehicle_data -> 504 (10314.362 ms)
teslamate_1  | 2021-08-16 19:03:48.672 car_id=1 [error] Error / :timeout
teslamate_1  | 2021-08-16 19:04:16.460 [info] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5/vehicle_data -> 408 (17786.190 ms)
teslamate_1  | 2021-08-16 19:04:25.119 car_id=1 [warn] Discarded incomplete fetch result
teslamate_1  | 2021-08-16 19:04:46.379 [warn] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5/vehicle_data -> 504 (11258.167 ms)
teslamate_1  | 2021-08-16 19:04:46.379 car_id=1 [error] Error / :timeout
teslamate_1  | 2021-08-16 19:05:05.634 [info] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5/vehicle_data -> 408 (9254.835 ms)
teslamate_1  | 2021-08-16 19:05:07.000 car_id=1 [warn] Discarded incomplete fetch result
teslamate_1  | 2021-08-16 19:05:26.215 [info] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5/vehicle_data -> 408 (9214.693 ms)
teslamate_1  | 2021-08-16 19:05:26.792 car_id=1 [warn] Discarded incomplete fetch result
teslamate_1  | 2021-08-16 19:05:46.177 [info] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5/vehicle_data -> 408 (9384.690 ms)
teslamate_1  | 2021-08-16 19:05:53.121 car_id=1 [warn] Discarded incomplete fetch result
teslamate_1  | 2021-08-16 19:06:12.953 [info] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5/vehicle_data -> 408 (9831.794 ms)
teslamate_1  | 2021-08-16 19:06:13.522 car_id=1 [warn] Discarded incomplete fetch result
teslamate_1  | 2021-08-16 19:06:32.683 [info] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5/vehicle_data -> 408 (9159.509 ms)
teslamate_1  | 2021-08-16 19:06:32.845 car_id=1 [warn] Discarded incomplete fetch result
teslamate_1  | 2021-08-16 19:06:52.948 [info] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5/vehicle_data -> 408 (10101.065 ms)
teslamate_1  | 2021-08-16 19:06:53.082 car_id=1 [warn] Discarded incomplete fetch result
teslamate_1  | 2021-08-16 19:07:12.242 [info] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5/vehicle_data -> 408 (9138.347 ms)
teslamate_1  | 2021-08-16 19:07:12.984 car_id=1 [warn] Discarded incomplete fetch result
teslamate_1  | 2021-08-16 19:07:32.510 [info] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5/vehicle_data -> 408 (9524.757 ms)
teslamate_1  | 2021-08-16 19:07:32.771 car_id=1 [warn] Discarded incomplete fetch result
teslamate_1  | 2021-08-16 19:07:52.860 [warn] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5/vehicle_data -> 504 (10087.728 ms)
teslamate_1  | 2021-08-16 19:07:52.861 car_id=1 [error] Error / :timeout
teslamate_1  | 2021-08-16 19:08:12.298 [info] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5/vehicle_data -> 408 (9436.095 ms)
teslamate_1  | 2021-08-16 19:08:12.475 car_id=1 [warn] Discarded incomplete fetch result
teslamate_1  | 2021-08-16 19:08:14.318 car_id=1 [info] Received stream data: :inactive
teslamate_1  | 2021-08-16 19:08:32.579 [warn] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5/vehicle_data -> 504 (10103.744 ms)
teslamate_1  | 2021-08-16 19:08:32.580 car_id=1 [error] Error / :timeout
teslamate_1  | 2021-08-16 19:08:55.208 [info] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5/vehicle_data -> 408 (12627.451 ms)
teslamate_1  | 2021-08-16 19:08:55.208 [warn] TeslaApi.Error / %{"error" => "https://mothership-api.vn.teslamotors.
com:443/vehicles/1052137850 => operation_timedout with 10s timeout", "error_description" => "", "response" => nil}
teslamate_1  | 2021-08-16 19:08:55.208 car_id=1 [error] Error / :unknown
teslamate_1  | 2021-08-16 19:16:53.341 [info] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5/vehicle_data -> 408 (11211.715 ms)
teslamate_1  | 2021-08-16 19:16:54.102 car_id=1 [warn] Discarded incomplete fetch result
teslamate_1  | 2021-08-16 19:17:20.894 car_id=1 [info] Fetch already in progress ...
teslamate_1  | 2021-08-16 19:17:21.651 car_id=1 [info] Fetch already in progress ...
teslamate_1  | 2021-08-16 19:17:22.395 car_id=1 [info] Fetch already in progress ...
teslamate_1  | 2021-08-16 19:17:22.636 car_id=1 [info] Fetch already in progress ...
teslamate_1  | 2021-08-16 19:17:24.133 car_id=1 [info] Fetch already in progress ...
teslamate_1  | 2021-08-16 19:17:25.657 [info] GET https://nominatim.openstreetmap.org/reverse -> 200 (323.732 ms)
teslamate_1  | 2021-08-16 19:17:26.155 [info] GET https://nominatim.openstreetmap.org/reverse -> 200 (374.369 ms)
teslamate_1  | 2021-08-16 19:17:26.163 car_id=1 [info] Driving / Ended / 21 km – 36 min
teslamate_1  | 2021-08-16 19:17:26.163 car_id=1 [info] Start / :online
teslamate_1  | 2021-08-16 19:21:42.515 [warn] GET https://owner-api.teslamotors.com/api/1/vehicles/4805709481152421
5/vehicle_data -> 504 (10095.617 ms)
teslamate_1  | 2021-08-16 19:21:42.516 car_id=1 [error] Error / :timeout
teslamate_1  | 2021-08-16 19:22:03.286 car_id=1 [info] Suspending logging
teslamate_1  | 2021-08-16 19:23:02.977 car_id=1 [info] Fetching vehicle state ...
teslamate_1  | 2021-08-16 19:36:57.113 car_id=1 [info] Fetching vehicle state ...
teslamate_1  | 2021-08-16 19:52:03.660 car_id=1 [info] Start / :asleep
teslamate_1  | 2021-08-16 19:52:03.665 car_id=1 [info] Disconnecting ..
```

**Comment by leewillis77** [comment]
Looks like a duplicate of #1883

**Comment by adriankumpf** [comment]
The Tesla Streaming API returned an unexpected error which caused the drive to be interrupted. This was fixed in 6c19a7b. To manually repair the affected drives please see [Manually fixing data – Terminate a drive or charge#](https://docs.teslamate.org/docs/maintenance/manually_fixing_data#terminate-a-drive-or-charge)
